templates/traas.yaml

```diff
@@ -156,7 +156,7 @@ resources:
         export HOME=/root
         BRANCH=${BRANCH#stable/}
         env STABLE_RELEASE=${BRANCH} tripleo-ci/scripts/tripleo.sh --repo-setup
-        yum -y install python-heat-agent*
+        yum -y install python-heat-agent* jq vim
```

Comment:
are these dependencies for the job itself? or just to make it useful while interactively debugging? if it's for the job itself, it should be in tripleo-quickstart itself.

Reply:
Just debugging.  jq is usually installed on overcloud like image but may be missing on other image.  vim, it's just that it's annoying to not have it.